### DISCUSSION TITLE
Welcome to infinity Discussions!

### DISCUSSION THREAD

**Discussion by michaelfeil** [michaelfeil/infinity#65] [Announcements]
Opening the discussion section! Feel free to discuss anything related.

**Comment by cduk** [comment]
Apologies for the dumb/question, but when would one use infinity vs TEI? I have a basic use case and am just looking to serve up an API where multiple internal clients can get GPU accelerated embeddings so I'm guessing both would work, but I'm interested to understand the use cases each lean towards.

**Reply by michaelfeil** [reply]
Would this be a good pointer: https://github.com/michaelfeil/infinity/issues/108
